src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import DropdownLink from "@/components/Dropdown/DropdownLink";
+
+import styles from "./Dropdown.module.scss";
+
+interface DropdownProps {
+  menu: {
+    osName: string;
+    icon: React.ReactNode;
+    links: string[];
+  };
+}
+
+function Dropdown({ menu: { osName, links, icon } }: DropdownProps) {
+  const [isOpen, setOpen] = useState(false);
+  const softwear = ["Metacon", "Conun Drive", "Conun Manager", "Download All"];
```

Comment:
there is a typo here whoops
Also further down where this is looped over

Reply:
> Let's see about splitting up that card component thingy

I did my best but it is still big! lol